md/builders.py

```diff
@@ -1,15 +1,91 @@
+from tempfile import NamedTemporaryFile
+
 import numpy as np
 from simtk import unit
 from simtk.openmm import app, Vec3
 
+from openeye.oechem import (
+    oemolistream,
+    oemolostream,
+    OEGraphMol,
+    OEReadMolecule,
+    OEWriteMolecule,
+    OEAltLocationFactory,
+)
+from openeye.oespruce import (
+    OEStructureMetadata,
+    OEMakeDesignUnitOptions,
+    OEMakeBioDesignUnits,
+)
+
+from ff.handlers.bcc_aromaticity import openeye_log_wrapper
+
+
+def prepare_protein(path: str, output_path: str):
+    """
+    Prepares protein
+
+    Performs the following preparation:
+    * Add missing hydrogens
+    * Remove everything except for the protein
```

Comment:
One thing I just realized is that it can also change the charge of the system. Not sure how to determine whether the charge assigned is better or worse. Will look at disabling it